src/component/footer.js

```diff
@@ -1,43 +1,25 @@
 import React from 'react';
-import Throttle from './footer-throttle';
+import {CacheButton} from './footer/cache';
+import Throttle from './footer/throttle';
+import {RequestCount} from './footer/requestCount';
 
 const {func, object} = React.PropTypes;
 
-export default class Footer extends React.Component {
+export const Footer = (props) => {
```

Comment:
The syntax `export default const ...` isn't valid, according to https://esdiscuss.org/topic/why-is-export-default-var-a-1-invalid-syntax.

Instead, you'd need to:
```
const Footer = (props) => {
...
}

export default Footer